src/models/Config.php

```diff
@@ -194,6 +194,7 @@ public function create(): bool
             ('s3_region', ''),
             ('s3_endpoint', ''),
             ('s3_verify_cert', '1'),
+            ('s3_path_style', '0'),
```

Comment:
I suggest naming the config `s3_use_path_style_endpoint` so it is the same as the actual option